core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -283,12 +283,11 @@ class WorkspaceService(
       }
       owners = ownersPolicy.map(policy => policy.memberEmails.map(_.value))
       canShare <- options.anyPresentFuture("canShare") {
-        accessLevel match {
-          case WorkspaceAccessLevels.Read  => Future.successful(false)
-          case WorkspaceAccessLevels.Owner => Future.successful(true)
-          case _ =>
-            val sharePolicy = SamWorkspaceActions.sharePolicy(accessLevel.toString.toLowerCase())
-            samDAO.userHasAction(SamResourceTypeNames.workspace, workspaceId, sharePolicy, ctx)
+        if (accessLevel < WorkspaceAccessLevels.Read) Future.successful(false)
```

Comment:
These are the original comparisons used to determine canShare.

Note: the only permission that is `< WorkspaceAccessLevels.Read` is `NoAccess`, which would have thrown an exception when getting the workspace earlier.

Reply:
Should there be a test case for PROJECT_OWNER?